config/initializers/airbrake.rb

```diff
@@ -1,7 +1,6 @@
-# This file is overwritten on deploy
-#
 Airbrake.configure do |config|
-  # Adding "production" to the development environments causes Airbrake not
-  # to attempt to send notifications.
-  config.development_environments << "production"
+  config.api_key = ENV['ERRBIT_API_KEY']
+  config.host = "errbit.#{ENV['GOVUK_APP_DOMAIN']}"
```

Comment:
`ERRBIT_HOST` doesn't currently exist anywhere. `errbit.#{ENV['GOVUK_APP_DOMAIN']}` is what's used everywhere else ([link](https://github.com/search?q=org%3Aalphagov+errbit.%23%7BENV%5B%27GOVUK_APP_DOMAIN%27%5D%7D&type=Code)), so it's probably best to keep that consistent.